gprofiler/main.py

```diff
@@ -760,6 +771,13 @@ def parse_cmd_args() -> configargparse.Namespace:
         " as well",
     )
 
+    parser.add_argument(
+        "--external-metadata",
+        default=None,
+        type=str,
+        help="Path to a file containing static & application metadata to be added to the profile",
```

Comment:
I'd add a word about that this flag is intended to be used mainly by other Granulates' components.
The rationale is that the average user won't "waste his time" trying to understand this feature, while this feature is not intended to be used by him.

Reply:
Done